salt/states/file.py

```diff
@@ -103,6 +142,11 @@ def run():
     value in single quotes. If the value is not wrapped in quotes it
     will be read by YAML as an integer and evaluated as an octal.
 
+.. _use-names-parameter:
+
+Use the names parameter to expand the contents of a single state
+----------------------------------------------------------------
+
 The ``names`` parameter, which is part of the state compiler, can be used to
```

Comment:
Aside: `names` should probably be in https://docs.saltproject.io/en/latest/ref/states/requisites.html

Reply:
I looked on that page, saw no dedicated section describing `names`.  Leaving as-is.

Reply:
That is the problem. It's a global state argument, not specific to `file`. It should be documented in a single place (that page) not every single state module.